src/main/java/net/testudobank/MvcController.java

```diff
@@ -395,10 +404,18 @@ public String submitDispute(@ModelAttribute("user") User user) {
       // If reversing a deposit puts customer back in overdraft
       if (reversalAmountInPennies > userBalanceInPennies){
         // check if the reversed deposit helped pay off overdraft balance
-        List<Map<String,Object>> overdraftLogs = TestudoBankRepository.getOverdraftLogs(jdbcTemplate, userID, (String)logToReverse.get("Timestamp"));
+        String dateTime = SQL_DATETIME_FORMATTER.format(convertLocalDateTime((LocalDateTime)logToReverse.get("Timestamp")));
```

Comment:
Maybe rename this variable to something like `datetimeOfReversedDeposit` for more clarity.